pkg/lwip/contrib/lwip.c

```diff
@@ -179,94 +230,106 @@ void lwip_bootstrap(void)
 {
     (void)_netif_add;   /* in case it is not used */
     /* TODO: do for every eligible netdev */
-#ifdef LWIP_NETIF_NUMOF
+#ifdef LWIP_NETIF_USED
+    int _netif = 0;
```

Comment:
Why the `_` prefix? It usually is used to denote private scope, but for a function variable basically all is private (to the function), so usually you use it for global variables and functions only.

Reply:
>  Why the _ prefix? It usually is used to denote private scope, but for a function variable basically all is private.

Yeah, of course I know that :wink: It was just to make sure to avoid potential name conflicts.

Reply:
Then 
```suggestion
    int netif_num = 0;
```
```suggestion
    int netif_count = 0;
```
or something of that kind?

Reply:
I renamed it to `netif_idx`.